saga-core/src/main/java/io/servicecomb/saga/core/SagaRequestImpl.java

```diff
@@ -38,14 +39,16 @@ public SagaRequestImpl(
       Transaction transaction,
       Compensation compensation,
       Fallback fallback,
-      String[] parents) {
+      String[] parents,
+      int failRetryDelayMilliseconds) {
 
     this.id = id;
     this.serviceName = serviceName;
     this.type = type;
     this.transaction = transaction;
     this.compensation = compensation;
     this.fallback = fallback;
+    this.failRetryDelayMilliseconds = failRetryDelayMilliseconds == 0 ? 50 : failRetryDelayMilliseconds;
```

Comment:
if delay is negative, thread.sleep will throw exception. we have to either throw now or use default delay too.

Reply:
OK